pkgs/development/python-modules/aioesphomeapi/default.nix

```
@@ -12,7 +12,7 @@
 
 buildPythonPackage rec {
   pname = "aioesphomeapi";
-  version = "10.14.0";
+  version = "11.0.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

See https://github.com/esphome/aioesphomeapi/releases/tag/v11.0.0

> Breaking Changes
> Bump min python to 3.9 (https://github.com/esphome/aioesphomeapi/pull/265) @jesserockz 

```suggestion
  disabled = pythonOlder "3.9";
```